### ISSUE TITLE
Getting Multiple tabs when opening an applet

### ISSUE THREAD

**Issue by gvc55** [MscrmTools/XrmToolBox#1283]
Not sure if I am missing a setting, but after the last XrmToolbox update, I am getting multiple tabs when I open a good number of applets (SQL4CDS, Plugin Tracer, etc.). It does not happen on the first applet, but if I open that same applet a second time, I get the three tabs. In case you are wondering, I am not double-clicking the applet icons. I am single clicking from the Recently Used Tools panel.

Please help on what I am missing here.

---
- Deployment: Online
- DB Version: 9.2.23083.204
- Connection Controls Version: 1.2023.6.56
- XrmToolBox Version: 1.2023.6.65

**Comment by MscrmTools** [comment]
You don't have to double click... I'm not sure if it's the issue. I never reproduced the issue so complicated to fix. 
Could you eventually provide a recording of the issue?

**Comment by gvc55** [comment]
I will try, if I can figure out how to record it. Will keep you posted.Sent from my iPhoneOn Oct 24, 2023, at 4:19 AM, MscrmTools ***@***.***> wrote:﻿
You don't have to double click... I'm not sure if it's the issue. I never reproduced the issue so complicated to fix.
Could you eventually provide a recording of the issue?

**Comment by gvc55** [comment]
Good afternoon. I did managed to get a recording but while doing that I discovered that the issue might be related to the Start Page link rather than the application itself. If the recording makes it to you (it's around 40 MB and not sure if it will get through), you will noticed that clicking on the Start Page link for an applet does generate 3 tabs. If I try the same applet from the "Tools" list, only one tab get added. Is there anything I can look at to see if there is something missing or out of wack in the Start Page?

https://github.com/MscrmTools/XrmToolBox/assets/64438054/3b5bcc7f-8677-4c7f-957c-a9d2de526966

**Comment by gvc55** [comment]
I think I found the problem. There were multiple Default connection files (3) which contain the same entries. My suspicion is that clicking on the Start Page link launched the applet for each of those connection files and thus created the multiple tabs. I removed the extra files and now only one tab is created. How the extra files got created in the first place is beyond me. Will keep an eye for the connections as see if they come back.
